docs/Quick-Start.md

```diff
@@ -60,12 +60,14 @@ $ ./bin/spark-shell --master local --jars ${carbondata_jar},${mysql_jar}
 import org.apache.spark.sql.CarbonContext
 import java.io.File
 import org.apache.hadoop.hive.conf.HiveConf
-val cc = new CarbonContext(sc, "./carbondata/store")
+val storePath = "hdfs://hacluster/Opt/CarbonStore"
```

Comment:
It is not mandatory to give hdfs path as store path. It can accept local path as well.